apps/fabric-website/src/components/App/App.tsx

```diff
@@ -49,25 +47,13 @@ export class App extends React.Component<IAppProps, any> {
     };
 
     return (
-      <Fabric
-        className='App'
-      >
-        <div className='App-wrapper'>
-          <div
-            className='App-nav'
-            style={ navStyle }
-          >
-            <Nav
-              pages={ AppState.pages }
-            />
+      <Fabric className="App">
+        <div className="App-wrapper">
+          <div className="App-nav" style={navStyle}>
+            <Nav pages={AppState.pages} />
           </div>
-          <div
-            className='App-content'
-            data-is-scrollable='true'
-            ref={ (el) => this._appContent = el }
-            data-app-content-div='true'
-          >
-            { this.props.children }
+          <div className="App-content" data-is-scrollable="true" ref={el => (this._appContent = el)} data-app-content-div="true">
```

Comment:
this looks less pretty; is it just because the line isn't long enough?

Reply:
I believe the answer is yes, I see the reverse correction later.

Reply:
Yes they recommend 80 character line width, we currently have 140 (i think for LPC we use 120)

What the prettier docs say is the following

> For readability we recommend against using more than 80 characters:
>
> In code styleguides, maximum line length rules are often set to 100 or 120. However, when humans write code, they don't strive to reach the maximum number of columns on every line. Developers often use whitespace to break up long lines for readability. In practice, the average line length often ends up well below the maximum.
>
> Prettier, on the other hand, strives to fit the most code into every line. With the print width set to 120, prettier may produce overly compact, or otherwise undesirable code.